### DISCUSSION TITLE
"KO7::" with "Kohana::"

### DISCUSSION THREAD

**Discussion by tomazov** [koseven/koseven#525] [General]
why did you replace **Kokhan::** with **KO7**, couldn’t it have been left?
these changes do not allow the code to be used normally
I'll have to rewrite all the code :-(

**Comment by uBizzy** [comment]
@tomazov, you have a module on KO7 to map all "Kohana::" classes. Just activate it and you are ready to go.

**Comment by timhj** [comment]
@uBizzy - Pretty sure I said the same thing when this was initially discussed but I have to agree with @tomazov,  now that it's been implemented and trying to work with the php8.3 branch this was not a great design decision. 

Trying to review historical changes in VS Code/Gitlens is effectively wiped because the original `Koseven::` versions of the files no longer exist.

I suggest for PHP8.4+ support implementing all the changes on Master to get it up and running.
